src/lib/content/model/ContentNotFoundError.ts

```diff
@@ -0,0 +1,8 @@
+export class ContentNotFoundError extends Error {
```

Comment:
Can you update this error to include the contentItem as a property, so users of the SDK don't have to parse the message.
For example
```js
const client = new ContentClient({
  hubName: 'myhub'
});

const slot = 'homepage-banner-slot';
client
  .getContentItemByKey(slot)
  .then(content => {
    console.log(content.body);
  })
  .catch(error => {
    if (error instanceof ContentNotFoundError) {
      console.wrror("Cannot find the content item: " + error.contentItem);
    } else {
      console.error("Unknown error", error);
  });
```

Reply:
Done.